src/PKSim.Core/Model/PopulationAnalyses/GroupingDefinition.cs

```diff
@@ -6,26 +6,11 @@
 
 namespace PKSim.Core.Model.PopulationAnalyses
 {
-   public interface IGroupingDefinition : IComparer<object>, IUpdatable
```

Comment:
interface IGroupingDefinition removed (there is also an abstract class GroupingDefinition)